cl/disclosures/tests.py

```diff
@@ -39,10 +44,20 @@ class DisclosureIngestionTest(TestCase):
         "JEF_format.pdf",
     )
 
+    @classmethod
+    def setUpTestData(cls) -> None:
+        judge = PersonWithChildrenFactory.create(id=2)
+        cls.test_disc = FinancialDisclosureFactory.create(
+            person=judge,
+        )
+        FinancialDisclosureFactory.create(
+            person=judge,
+        )
+
     def test_financial_disclosure_ingestion(self) -> None:
         """Can we successfully ingest disclosures at a high level?"""
 
-        test_disclosure = FinancialDisclosure.objects.get(pk=1)
+        test_disclosure = FinancialDisclosure.objects.get(pk=self.test_disc.id)
```

Comment:
Can't you just do:
```suggestion
        test_disclosure = self.test_disc
```
?

Reply:
yes... what a weird choice.

Reply:
I think there is a cleaner thing to do here so im going to not use your suggestion.  but I will fix it